ballerina-shell/modules/shell-cli/src/main/resources/command.help.txt

```diff
@@ -0,0 +1,22 @@
+Ballerina Shell Help
+
+Type a ballerina statement, expression or a declaration.
+
+Imports should be done with prefixes and the modules
+should be available in the classpath.
+To see available imports use /imports.
+
+Following commands are the in-built commands.
```

Comment:
Is it in-built or `builtin commands`? We are mostly using builtin keyword.

Reply:
I rename to built-in commands